content/en/news/releases/1.19.x/announcing-1.19/change-notes/index.md

```diff
@@ -0,0 +1,301 @@
+
+- **Added** support for a flag called `USE_EXTERNAL_WORKLOAD_SDS`. When set to true, it will require an external SDS workload socket and it will prevent the istio-proxy from starting if the workload SDS socket is not found.
+  ([Issue #45534](https://github.com/istio/istio/issues/45534))
+
+
+- **Fixed** an issue where `jwk` issuer was not resolved correctly when having a trailing slash in the issuer URL.
+  ([Issue #45546](https://github.com/istio/istio/issues/45546))
+
+- **Removed** the `SPIFFE_BUNDLE_ENDPOINTS` feature flag.
+
+## Telemetry
+
+- **Added** new metric named `provider_lookup_cluster_failures` for lookup cluster failures.
+
+- **Added** support for K8s controller queue metrics, enabled by setting env variable `ISTIO_ENABLE_CONTROLLER_QUEUE_METRICS` to `true`.  ([Issue #44985](https://github.com/istio/istio/issues/44985))
+
+- **Added** a flag to disable `OTel` builtin resource labels.
+
+- **Added** `cluster` label for `remote_cluster_sync_timeouts_total` metric.  ([Issue #44489](https://github.com/istio/istio/issues/44489))
+
+- **Added** support for annotation `sidecar.istio.io/statsHistogramBuckets` to customize the histogram buckets in the proxy.
+
+- **Added** HTTP metadata exchange filter to support a fallback to xDS workload metadata discovery in addition to the metadata HTTP headers. The discovery method is off by default.
+
+- **Added** an option to configure Envoy to report load stats to the Load Reporting Service (LRS) server.
+
+- **Fixed** an issue where disabling a log provider through Istio telemetry API would not work.
+
+- **Fixed** an issue where `Telemetry` would not be fully disabled unless `match.metric=ALL_METRICS` was explicitly specified; matching all metrics is now correctly considered as the default.
+
+## Extensibility
+
+- **Added** an option to fail open on fetch failure and VM fatal errors.
+
+## Installation
+
+- **Improved** usage on OpenShift clusters by removing the need to manually create a `NetworkAttachmentDefinition` resource in every application namespace.
+
+- **Updated** Kiali addon to version `v1.72.0`.
+
+- **Added** support for `PodDisruptionBudget` (PDB) in the Gateway chart.
+  ([Issue #44469](https://github.com/istio/istio/issues/44469))
+
+- **Added** the Helm value of setting CNI ambient `configDir` path.
+  ([Issue #45400](https://github.com/istio/istio/issues/45400))
+
+- **Added** `amd64` named artifacts for macOS and Windows. The `amd64` flavor of the artifacts did not contain the architecture in the name as we do for the other operating systems. This makes the artifact naming consistent.
+
+**Deprecated** the macOS and Windows artifacts without an architecture specified in the name
+(ex: `istio-1.18.0-osx.tar.gz`). They will be removed in several releases. They have been replaced
+by artifacts containing the architecture in the name (ex: `istio-1.18.0-osx-amd64.tar.gz`).  ([Issue #45677](https://github.com/istio/istio/issues/45677))
```

Comment:
Here is the deprecated item. If we leave it here, we should make it a bullet like the rest.